services/workflows-service/prisma/schema.prisma

```diff
@@ -155,6 +155,7 @@ model WorkflowDefinition {
 
   definitionType     String
   definition         Json
+  transitionSchema   Json?
```

Comment:
Why here and not inside the definition, near the transition itself? it will make us manage it in different places and can be hard to track